source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -1,143 +1,139 @@
+- ensuring the wrong people do not have access
+
+You must ensure that when people leave GDS, their access is revoked or expired. You should also identify who did what and when (audit), this has varying importance depending on its context.
+
+Please note, Technical limitations may prevent a full audit from being achieved, or may only provide a partial audit.
+
+### Account management principles
+
+To achieve the above goals, there are 3 principles we recommend you follow:
+
+- minimising the number of secrets an individual needs to keep reduces the number of credentials to revoke when someone leaves GDS. Passwords are particularly problematic secrets. Other forms of secrets, for example secret access keys or OAuth tokens, are more easily rotated
+
+- no individual should ever have to use credentials associated with
+  another individual as this counters the audit goal
+
+- There should be no need to share credentials between service teams. This creates problems for audit and for knowing when credentials should be revoked, although technical limitations may mean this is sometimes unavoidable
 
 ## Creating accounts for shared third-party services
 
-The recommendations here depend on whether the service supports
-organisation or team access natively.
+Our recommendations in this section depend whether a third-party service natively supports organisation or team level access.
 
-### Services that support organisation accounts
+### Services supporting organisation accounts
 
 For services supporting organisation accounts (for example, the use of alphagov on github) you should use them to ensure only the appropriate individuals on your team have access.
 
-When creating your accounts, we recommend you:
+When creating accounts, we recommend you:
 
-- Create separate accounts for each individual rather than sharing
+- create separate accounts for each individual rather than sharing
   credentials
-- Prefer using a single-sign-on provider over creating an account with
+- use a single-sign-on provider over creating an account with a
   separate username and password
-- Prefer using G Suite as a single sign-on provider over any other
-  provider
-    - this is because G Suite is a core part of the leaver's process and
+- use G Suite as a single sign-on provider over other
+  providers
+    - G Suite is a core part of the GDS leaver's process,
       so we have high confidence this will be revoked when somebody
       leaves
     - Sometimes GitHub may be preferable as sign-on provider, for
       services that can use GitHub’s team structure to manage
-      permissions. For example, Travis CI does this.
-- Ensure that you have a documented process for removing individual's
-  access when they leave.
-    - If you use G Suite then their access will be revoked when their
-      G Suite access is revoked.  However, you may still wish to clean
-      up the individual's account with the third-party service.
+      permissions. For example, travis does this
```

Comment:
looks like you accidentally undid the change from 8e9157ea5d34ac8dc679cd8e78b46768b1d7f550 here